apps/charterafrica/src/payload/collections/Tools.js

```diff
@@ -0,0 +1,307 @@
+import {
+  TOOL_COLLECTION,
+  ORGANIZATION_COLLECTION,
+  CONTRIBUTORS_COLLECTION,
+} from "../../lib/ecosystem/models";
+import donors from "../fields/donorsField";
+import slug from "../fields/slug";
+import source from "../fields/toolSourceField";
+import updatedAt from "../fields/updatedAt";
+
+const Tools = {
+  slug: TOOL_COLLECTION,
+  admin: {
+    useAsTitle: "externalId",
+    defaultColumns: ["externalId", "name", "location", "description"],
```

Comment:
```suggestion
    defaultColumns: ["externalId", "name", "location", "source"],
```

`description` is too long to appear in a column